src/v/bytes/random.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2023 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+#pragma once
+
+#include "bytes/bytes.h"
+#include "bytes/iobuf.h"
+
+namespace random_generators {
+
+bytes get_bytes(size_t n = 128 * 1024);
```

Comment:
In our effort to make sure stuff in headers have documentation should we add a comment here?

Reply:
100%. But I'd like to draw a line in the sand and say that header / dependency reorganization can't be blocked on documenting interfaces that aren't materially changing. I mean, it'd be easy to do in the case of get_bytes, but for the vast majority of the reorg I don't think that'll hold.

Reply:
Perhaps what we should do as part of the reorg process is generate a number of documentation improvement tasks that anyone can pick up?

Reply:
Yeah SG to me. Maybe we just create a docs issue and create a task list of methods that need docs?